<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPopulasi</class>
 <widget class="QWidget" name="FormPopulasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>331</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="idPopulasiLabel">
      <property name="text">
       <string>ID Populasi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idPopulasiLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="tanggalPopulasiLabel">
      <property name="text">
       <string>Tanggal Populasi</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="tanggalPopulasiDateEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="jumlahMatiLabel">
      <property name="text">
       <string>Jumlah Mati</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="jumlahMatiLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="jumlahHidupLabel">
      <property name="text">
       <string>Jumlah Hidup</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="jumlahHidupLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="sisaLabel">
      <property name="text">
       <string>Sisa</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="sisaLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="kodeKandangLabel">
      <property name="text">
       <string>KodeKandang</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="kodeKandangComboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>269</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
